hanbatbox-api/src/main/java/com/hbu/hanbatbox/module/S3/controller/S3Controller.java

```diff
@@ -0,0 +1,46 @@
+package com.hbu.hanbatbox.module.S3.controller;
+
+import com.hbu.hanbatbox.module.S3.service.S3Service;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequestMapping("/api/s3")
+public class S3Controller {
+
+  private final S3Service s3Service;
+
+  @Autowired
+  public S3Controller(S3Service s3Service) {
+    this.s3Service = s3Service;
+  }
```

Comment:
```suggestion

@Controller
@RequiredArgsConstructor
public class S3Controller {

  private final S3Service s3Service;
```

코드는 다음처럼도 표현할 수 있습니다.
이점은 다음과 같습니다.
* `@Autowired` 로 스프링 종속적인 코드에서 `Plain Of Java Object` 로 변경할 수 있음 (의존성이 낮아짐)
* 더 간결한 표현이 가능함

Reply:
확인했습니다. 바로 적용할게요